transloadit-rails.gemspec

```diff
@@ -15,7 +15,8 @@ Gem::Specification.new do |gem|
   gem.summary     = 'Official Rails gem for Transloadit'
   gem.description = 'The transloadit-rails gem allows you to automate uploading files through the Transloadit REST API'
 
-  gem.required_rubygems_version = '>= 1.3.6'
+  gem.required_rubygems_version = '>= 2.2.0'
```

Comment:
IMHO This should say 2.1.0, as this is also what we're running tests against on Travis :)